libiocage/lib/errors.py

```diff
@@ -22,17 +22,27 @@
 # IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+# MyPy
+import libzfs  # noqa: F401
```

Comment:
huh?

Reply:
mypy complained about missing libzfs, so I've added it to the imports. Flake8 would now complain about the unused import unless we ignore F401.